client/state/data-layer/wpcom/comments/test/index.js

```diff
@@ -97,6 +99,74 @@ describe( 'wpcom-api', () => {
 			} );
 		} );
 
+		describe( '#writePostComment()', () => {
+			const action = {
+				type: 'DUMMY',
+				siteId: 101010,
```

Comment:
This is our current convention 😄 It helps guard against an accidental integration test ;)